### PR TITLE
add image data figure, 2 min CTD stop by Hahn

### PR DESCRIPTION
**Pull request by soerenthomsen**
This PR implements the discussion #114 based on Hahn et al. 2014 and the associated PhD thesis by @hahn-johannes 

I further removed some old figure text which was already commented out.

### CODE REVIEW

**Comment by soerenthomsen** [comment]
This PR closes #114

**Comment by soerenthomsen** [comment]
We might want to improve the figure caption. Any input is more than welcome. 

Please review @tomhull @hahn-johannes and @patricialg

File: sections/oxygen_mission_execution.md [hunk 1]
```diff
@@ -79,16 +75,18 @@ The combined data collected following these (CTD and lab calibration) steps will
...
--->
-
 :::{figure-md} datalogger
 <img src="../images/Datalogger.png" alt="Data logger on CTD" class="bg-primary mb-1" width="600px">
 
 GEOMAR oxygen data loggers (Aanderaa Optode mounted on data logger) attached to a CTD frame and prepared for in-situ calibration during a CTD cast. Panels in the upper row show fixation with straps and zip ties. Panel in the lower row shows fixation with scaffolding clamps and tape in the interior lower part of the CTD frame.
 :::
 
+:::{figure-md} CTD_calibration_stop
+<img src="../images/CTD_calibration_01_03.png" alt="CTD calibration stop full cast" class="bg-primary mb-1" width="600px">
```
Comment (tomhull) [inline sections/oxygen_mission_execution.md:85]: Was this figure copied from Johannes's thesis or created for the SOP? There is a a black shaded region which doesn't look great.

Comment (soerenthomsen) [reply sections/oxygen_mission_execution.md:85]: @hahn-johannes prepared the figure for the SOP. So no copy right issues here. 

I will try to do better in combining both png's. If anyone has good tools let me know.

Comment (soerenthomsen) [reply sections/oxygen_mission_execution.md:85]: I uploaded a new merged version.

**Pull request merged**